opre_ops/opre_ops/settings/common.py

```diff
@@ -25,6 +25,7 @@
 # Application definition
 
 INSTALLED_APPS = [
+    'opre_ops.apps.OpsSiteConfig',
```

Comment:
[Question] [the order of `INSTALLED_APP` matters](https://docs.djangoproject.com/en/3.2/ref/templates/api/#django.template.loaders.app_directories.Loader), should this addition be at the top?

Reply:
This is the order the documentation had the installed apps in, but I don't think it matters.